tests/cql/CqlAggregateFunctionsTest.xml

```diff
@@ -68,13 +72,23 @@
 			<expression>AnyTrue({})</expression>
 			<output>false</output>
 		</test>
+		<test name="AnyTrueIsFalseWhenNull">
+			<expression>AnyTrue(null)</expression>
+			<output>false</output>
+		</test>
 	</group>
 	<group name="Avg">
 		<test name="AvgTest1">
 			<expression>Avg({ 1.0, 2.0, 3.0, 6.0 })</expression>
 			<output>3.0</output>
 		</test>
 	</group>
+	<group name="Product">
+		<test name="Product_Long">
```

Comment:
nit: I don't think underscores are used anywhere else for test names in this repo, so consider remaining consistent 

```suggestion
		<test name="ProductLong">
```

I personally don't feel that strongly though.